app/src/main/res/layout/item_creating_right_convention.xml

```diff
@@ -64,11 +71,12 @@
             android:layout_width="@dimen/dp_24"
             android:layout_height="@dimen/dp_24"
             android:layout_marginBottom="@dimen/dp_8"
-            android:layout_marginEnd="@dimen/dp_8"
             android:layout_marginTop="@dimen/dp_8"
-            android:background="@drawable/ic_audio_conversation"
+            android:background="@drawable/bg_menu_media_creating_conversation"
             android:contentDescription="@string/description_recording_audio_convention"
-            android:onClick="@{() -> viewModel.onAudioConversationClick(conversations)}"
+            android:onClick="@{(v) -> viewModel.showMenu(v, position, conversations)}"
+            android:padding="@dimen/dp_4"
+            android:src="@drawable/ic_menu_media"
```

Comment:
ERROR: [VectorDrawableCompat] When using VectorDrawableCompat, you need to use `app:srcCompat`.
 To use VectorDrawableCompat, you need to make two modifications to your project. First, set `android.defaultConfig.vectorDrawables.useSupportLibrary = true` in your `build.gradle` file, and second, use `app:srcCompat` instead of `android:src` to refer to vector drawables.